runtime/doc/usr_27.txt

```diff
@@ -83,7 +83,7 @@ matter if 'ignorecase' or 'smartcase' was changed.
 
 	Note:
 	If your search takes much longer than you expected, you can interrupt
-	it with CTRL-C on Unix and  CTRL-Break on MS-DOS and MS-Windows.
+	it with CTRL-C on Unix and CTRL-Break on Windows.
```

Comment:
Same thing about `CTRL-Break`. Not sure. Although this would seem to indicate it works both on DOS and Windows... Still think `CTRL-C` should work though...